inductiva/tasks/task.py

```diff
@@ -192,15 +192,15 @@ def set_default_output_files(self,
 
     def get_output(
         self,
-        full_output: bool = True,
+        full: bool = False,
```

Comment:
I feel that the default here should be `True`, since most tasks won't be associated with a scenario that controls which output files should be downloaded by default.

Reply:
With default to true, the name needs to be changed, otherwise it does not make sense to call "full=False".